src/lsp/MessageHandlers.re

```diff
@@ -253,29 +253,93 @@ let handlers: list((string, (state, Json.t) => result((state, Json.t), string)))
       }
     })
   }),
-
-  ("textDocument/rangeFormatting", (state, params) => {
+  
+  ("textDocument/rangeFormatting", fun (state, params) => {
```

Comment:
why the `fun`? That's deprecated

Reply:
I have no ideas I did I write that. Probably some old muscle memory